src/how-tos/deploy-operate.md

```diff
@@ -1,41 +1,130 @@
+You will need to download the YAML manifest for the ingress controller. You can find an example on the [Ingress Nginx Quickstart guide](https://kubernetes.github.io/ingress-nginx/deploy/#quick-start)
 
-```output:no-line-numbers
-// default Zilla config
-name: default
+```bash:no-line-numbers
+curl https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.10.1/deploy/static/provider/cloud/deploy.yaml > ingress-deploy.yaml
+```
+
+Once you have the Ingress Nginx YAML manifest, you must add the TCP port proxies for the ingress controller to allow your ports to pass through.
+
+Here is how to add ports `7183` and `7151` to the `service/ingress-nginx-controller`.
+
+```yaml{10-17}
+kind: Service
+metadata:
+...
+  name: ingress-nginx-controller
+  namespace: ingress-nginx
+spec:
+...
+  ports:
+...
+ - name: proxied-tcp-7183
+    port: 7183
+    targetPort: 7183
+    protocol: TCP
+ - name: proxied-tcp-7151
+    port: 7151
+    targetPort: 7151
+    protocol: TCP
+...
+```
+
+Finally, we need to configure the Ingress Nginx controller to look for port mappings in the `tcp-services` by adding the `--tcp-services-configmap=$(POD_NAMESPACE)/tcp-services` argument to the Deployment container args.
+
+```yaml{9}
+kind: Deployment
+spec:
+  template:
+    spec:
+      containers:
+ - args:
+  - /nginx-ingress-controller
+...
+  - --tcp-services-configmap=$(POD_NAMESPACE)/tcp-services
+```
+
+Create the ingress controller:
+
+```bash:no-line-numbers
+kubectl apply -f ingress-deploy.yaml
+```
+
+The ingress controller will allow your ports to pass through, and you can configure which services should receive the requests made at those ports.
+
+### Get diagnostics from zilla pods
 
-// Zilla status
-started
+For every running zilla pod you will need to first copy the `/var/run/zilla` directory to make sure no additional files are written while it is compressed then compress the full directory to make it easier to copy.
+
+```bash:no-line-numbers
+kubectl get pod \
+-l "app.kubernetes.io/name=zilla" \
+-n $NAMESPACE \
+--field-selector=status.phase=Running \
+-o custom-columns=name:metadata.name --no-headers \
+| xargs -I{} kubectl exec {} -n $NAMESPACE -c zilla -- sh -c "cp -r /var/run/zilla /tmp/zilla && tar czf /tmp/zilla.tar.gz /tmp/zilla && rm -rf /tmp/zilla"
 ```
 
-Specify your own `zilla.yaml` file.
+Copy the compressed `/var/run/zilla` directory off of the pod into your local directory using the pod name. 
 
 ```bash:no-line-numbers
-docker run -v ./zilla.yaml:/etc/zilla/zilla.yaml ghcr.io/aklivity/zilla:latest start -v
+kubectl get pod \
+-l "app.kubernetes.io/name=zilla" \
+-n $NAMESPACE \
+--field-selector=status.phase=Running \
+-o custom-columns=name:metadata.name --no-headers \
+| xargs -I{} kubectl cp  -n $NAMESPACE {}:/tmp/zilla.tar.gz ./{}.tar.gz
 ```
 
+Now you have a copy of the Zilla runtime directory for each running pod. This information can be used to diagnose all of the traffic zilla has managed.
+
```

Comment:
Do we want to add a suggestion to include this when filing issues or that it may be asked for by the zilla team to help diagnose an issue when it is reported?

Reply:
Yeah if you like this I think adding a link to this section in the github bug template would be good. I was planning to make that change after we release this.